packages/lesswrong/components/common/ExpandableSection.tsx

```diff
@@ -75,10 +79,17 @@ const ExpandableSection = ({
             </div>
           }
         >
-          {expanded && afterTitleTo &&
-            <Link to={afterTitleTo} className={classes.afterTitleLink}>
-              {afterTitleText}
-            </Link>
+          {expanded && (AfterTitleComponent || afterTitleTo) &&
+            <div className={classes.afterContainer}>
+              {AfterTitleComponent &&
+                <AfterTitleComponent />
+              }
+              {afterTitleTo &&
+                <Link to={afterTitleTo}>
+                  {afterTitleText}
+                </Link>
+              }
+            </div>
```

Comment:
These things get a bit squished on small screens, and then the checkbox is hidden entirely on the smallest screens.
<img width="353" alt="Screen Shot 2023-10-16 at 3 53 41 PM" src="https://github.com/ForumMagnum/ForumMagnum/assets/9057804/b4453f2f-82a0-4bc6-aaa5-e276e04f2eae">

Probably the "Show community" checkbox is important to display on all screen sizes, but the "View more" is ok to lose on small screens. Maybe we just make the checkbox wrap onto a second line.

Reply:
Also unfortunately things are not quite aligned in Safari:

<img width="679" alt="Screen Shot 2023-10-16 at 4 19 06 PM" src="https://github.com/ForumMagnum/ForumMagnum/assets/9057804/d645e7c8-0f37-42cb-9980-d8a5f07559c1">

Reply:
Thanks so much for catching these 😮 

> Probably the "Show community" checkbox is important to display on all screen sizes, but the "View more" is ok to lose on small screens. Maybe we just make the checkbox wrap onto a second line.

I would probably guess the opposite since the community checkbox is more of a one-off thing to do, whereas you may want to view more more often (not strongly held though). So I'd probably hide the checkbox instead

Reply:
Yeah I'm not really sure which is best. I was thinking that there is still a "Load more" button so you can still see more quick takes that way on mobile. But if you are just a mobile user, you might not realize that community quick takes are being filtered out at all without the checkbox.